src/eduid_webapp/email/views.py

```diff
@@ -184,9 +182,22 @@ def verify(user, code, email):
             '_status': 'error',
             'message': 'emails.code_expired_send_new'
         }
-
+    if code != state.verification.verification_code:
```

Comment:
Hmm, when you now check for a matching code _after_ you check for timeout, a new code will be sent out even if the wrong code is supplied. I'm a bit unsure what is possible to do through the UI, but it appears incorrect to check timeout before checking the code.